webpack-custom.config.js

```diff
@@ -0,0 +1,12 @@
+const ArcGISPlugin = require('@arcgis/webpack-plugin');
+/**
+ * Configuration items defined here will be appended to the end of the existing webpack config defined by the Angular CLI.
+ */
+module.exports = {
```

Comment:
This is pretty cool

Reply:
I'm not sure that it's going to work. :( I can build the application but not the library by itself. After looking at [the docs](https://angular.io/cli/build), it looks like library builds use something other than webpack? When I try to run `ng build layer-selector` I get `Unknown option: '--project'`.

Reply:
bummer...